src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -827,3 +827,15 @@
     text: >
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
+
+helps['acr check-health'] = """
+type: command
+short-summary: Gets health information on the environment and, optionally, a target registry.
+examples:
+  - name: Gets health state with target registry 'MyRegistry', skipping confirmation for pulling image.
+    text: >
+        az acr check-health -n myRegistry
+  - name: Gets health state with target registry 'myRegistry', without stopping on first error.
+    text: >
+        az acr check-health -n myRegistry --ignore
```

Comment:
As per `params` file, the argument is `--ignore-errors` ?

Reply:
Yes :) fixed it, thank you!